mesheryctl/internal/cli/root/system/config.go

```diff
@@ -85,6 +100,95 @@ func getContexts(configFile, tokenPath string) ([]string, error) {
 	return contexts, nil
 }
 
+// Tries to get AWS user credentials required for out of cluster deployments
+// to work.
+// Firstly looks for them in environment variables AWS_ACCESS_KEY_ID
+// and AWS_SECRET_ACCESS_KEY
+// Secondly in the shared directory at $HOME/.aws/
+// Lastly tries to take user input and generates a file in ~/.meshery/aws
+// TODO: Support EC2 instance role credentials
+// https://pkg.go.dev/github.com/aws/aws-sdk-go@v1.40.43#readme-configuring-credentials
+func awsCredentialHelper() error {
+	log.Info("Looking for aws user credentials in $AWS_ACCESS_KEY_ID and $AWS_SECRET_ACCESS_KEY OR ~/.aws/credentials...")
+	sess := session.Must(session.NewSession())
+	creds := sess.Config.Credentials
+	credValue, err := creds.Get()
+
+	if err != nil {
+		log.Info("Credentials not found. Generating static credentials now...")
```

Comment:
Would it be a better to elaborate? i.e.,
Credentials not found in environment variables. Generating static credentials now..

Reply:
yes :heavy_check_mark: